### ISSUE TITLE
run_model_bpa_bulk execution fails

### ISSUE THREAD

**Issue by plorenc** [microsoft/semantic-link-labs#277]
**Describe the bug**
**run_model_bpa_bulk** execution fails no matter what attribute for **workspace** parameter is provided.

**To Reproduce** 
Run the **run_model_bpa_bulk** function with **workspace**=None or any existing Workspace

**Log**
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[13], line 1
----> 1 labs.run_model_bpa_bulk(workspace=None)

File ~/cluster-env/clonedenv/lib/python3.11/site-packages/sempy/_utils/_log.py:348, in mds_log.<locals>.get_wrapper.<locals>.log_decorator_wrapper(*args, **kwargs)
    345 start_time = time.perf_counter()
    347 try:
--> 348     result = func(*args, **kwargs)
    350     # The invocation for get_message_dict moves after the function
    351     # so it can access the state after the method call
    352     message.update(extractor.get_completion_message_dict(result, arg_dict))

File ~/cluster-env/clonedenv/lib/python3.11/site-packages/sempy_labs/_model_bpa_bulk.py:104, in run_model_bpa_bulk(rules, extended, language, workspace, skip_models, skip_models_in_workspace)
    101 dfD = fabric.list_datasets(workspace=wksp, mode="rest")
    103 # Skip models in workspace
--> 104 skip_models_wkspc = skip_models_in_workspace.get(wksp)
    105 dfD = dfD[~dfD["Dataset Name"].isin(skip_models_wkspc)]
    107 # Exclude default semantic models

AttributeError: 'NoneType' object has no attribute 'get'
```

Version 0.8.5

**Comment by m-kovalsky** [comment]
Thanks for letting me know. I will fix this in a quick release.

**Comment by m-kovalsky** [comment]
Fixed in [0.8.6](https://github.com/microsoft/semantic-link-labs/releases/tag/0.8.6).

**Comment by plorenc** [comment]
Tested - it's working!
Thank you!
